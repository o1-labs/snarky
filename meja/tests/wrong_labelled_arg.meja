let a = fun (~a) => { a; };

let b = fun () => { a(~b = true); };
