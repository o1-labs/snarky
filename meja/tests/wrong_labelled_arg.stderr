File "tests/wrong_labelled_arg.meja", line 3, characters 27-31:
Error: Incompatible types a:'a -> 'a and b:'b -> 'c:
       Cannot unify a:'a -> 'a and b:'b -> 'c
